### PR TITLE
Several db columns removed or widened.

### PR DESCRIPTION
**Pull request by tft-kyrre**
Fara database cleanup.

### CODE REVIEW

File: src/Services/FaraImporter.php [hunk 1]
```diff
@@ -54,6 +69,18 @@ public function import(string $file)
...
+    // Remove column names as specified by the columnFilter array without re-
+    // indexing the returned array.
+    protected function getFilteredColumns(string $tableName, array $columns): array
+    {
+        if (!array_key_exists($tableName, $this->columnFilter)) {
+            // No column filter available for this table.
+            return $columns;
+        }
+        $filter = $this->columnFilter[$tableName];
+        return array_filter($columns, fn($col) => !in_array($col, $filter));
```
Comment (imable) [inline src/Services/FaraImporter.php]: If we rename the filter variable, it is slightly clearer what is happening :)

```suggestion
        $ignoreCols = $this->columnFilter[$tableName];
        return array_filter($columns, fn($col) => !in_array($col, $ignoreCols));
```

**Review by tft-abel — APPROVED** [review]

**Review by tfk-kaare — COMMENTED** [review]
Instead of using array_filter and custom methods for this, you can use array_diff* or array_intersect* with or without keys as comparator. They are more efficient and reduces the code :-)

**Comment by tft-kyrre** [comment]
"Simplified", but it didn't really reduce the code size a lot.

**Review by tfk-kaare — APPROVED** [review]
I see …

**Pull request merged**